proto/evmos/revenue/v1/tx.proto

```diff
@@ -21,6 +24,11 @@ service Msg {
   rpc CancelRevenue(MsgCancelRevenue) returns (MsgCancelRevenueResponse) {
     option (google.api.http).post = "/evmos/revenue/v1/tx/cancel_revenue";
   };
+  // UpdateParams defined a governance operation for updating the x/revenue module parameters.
+  // The authority is hard-coded to the Cosmos SDK x/gov module account
+  rpc UpdateParams(MsgUpdateParams) returns (MsgUpdateParamsResponse) {
+    option (google.api.http).post ="/evmos/revenue/v1/tx/update_params";
```

Comment:
Q: Considering that the update params is done via gov proposal, do we need to expose this as an HTTP endpoint? Same question applies for other modules